logexport/deserialize.py

```diff
@@ -72,8 +76,14 @@ def get_timestamp(load: dict) -> str | None:
     )
 
 
-def create_labels_string(category: str | None, type: str | None) -> str:
+def create_labels_string(
+    category: str | None, type: str | None, addional_labels: dict[str, str]
+) -> str:
```

Comment:
just figured, if one label key or value has a `,` or a `{}` character, it breaks the serialized labels defintion right?

Reply:
Good catch! I'll escape it.